src/OrchardCore.Modules/OrchardCore.Menu/Views/MenuItemLink.cshtml

```diff
@@ -1 +1 @@
-﻿<a href="@(Model.Href ?? "#")">@Model.Text</a>
+<a href="@(Model.Href ?? "#")" @(!string.IsNullOrEmpty(Model.Target) ? $"target=\"" + Model.Target + "\"" : string.Empty)>@Model.Text</a>
```

Comment:
You can always add target attribute. Use _self as the default value

Reply:
If you use the default values, there will be only one option here.
![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/649b9d9a-1f52-43ed-871a-78bfe393f184)

![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/d5eda91e-d50c-48d3-bede-f0adfdf670f5)